src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -1320,11 +1345,9 @@ namespace OpenMS
       {
         current_peptide_.setMetaValue("pepxml_spectrum_name", native_spectrum_name_);
       }
-      if (search_engine_ == "Comet")
-      {
-        current_peptide_.setMetaValue("spectrum_reference", native_spectrum_name_);
-        //TODO: we really need something uniform here, like scan number - and not in metainfointerface
-      }
+      //TODO: we really need something uniform here, like scan number - and not in metainfointerface
```

Comment:
not sure if scan number is better than e.g. native id once we also deal with ion mobility frames etc.

Reply:
Just copied. But the main gist is true. It should be a fixed field.